man/systemd.service.xml

```diff
@@ -96,9 +96,12 @@
     <varname>After=</varname> on
     <filename>dbus.socket</filename>.</para>
 
-    <para>Socket activated service are automatically ordered after
+    <para>Socket activated services are automatically ordered after
     their activated <filename>.socket</filename> units via an
-    automatic <varname>After=</varname> dependency.</para>
+    automatic <varname>After=</varname> dependency.
+    Services also pull in all <filename>.socket</filename> units
+    listed in <varname>Sockets=</varname> via an automatic
+    <varname>Wants=</varname> (and <varname>After=</varname>) dependency.</para>
```

Comment:
I'd probably drop the brackets and just say "… via automatic Wants= and After= dependencies."